main.py

```diff
@@ -95,6 +96,28 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+previous_queue = {}
+current_queue = {}
+@app.get("/experiment/queue/", response_model=Dict[int, Dict[str, Any]])
```

Comment:
Please add two empty lines after defining `current_queue`.

Reply:
Also, as I know, the `response_model` is only necessary when it returns a Pydantic model.
Please check it!